pkg/apis/duck/v1alpha1/subscribable_types.go

```diff
@@ -109,7 +109,7 @@ type SubscribableTypeSpec struct {
 
 // SubscribableTypeStatus shows how we expect folks to embed Subscribable in their Status field.
 type SubscribableTypeStatus struct {
-	SubscribableStatus *SubscribableStatus `json:"subscribablestatus,omitempty"`
+	SubscribableStatus *SubscribableStatus `json:"subscribableStatus,omitempty"`
```

Comment:
So this would be 100% correct if we hadn't already released this code. Sadly we have, so we need to be careful to ensure backward compatibility.

I think the way to do this in a backwards compatible way is to:

1. Create two fields here, one `subscribablestatus` and one `subscribableStatus`. These are exact duplicates of each other.
2. Change all the code writing to this field to write both fields (they are exact copies of each other).
3. Change all the code reading this field to prefer reading the new field. If the new field is not present, then read the old one.

All of this should be done in a single PR.

Wait a full release.
1. Delete the old field and all code reading and writing it.

This process guarantees that it is safe to upgrade from 0.N to 0.N+1 and from 0.N+1 to 0.N+2. The 0.N+1 code will initially read the old field, as the new one is not yet set. Upgrading the controllers will force a reconcile of all objects, so the new field will be written. Then the upgrade from 0.N+1 to 0.N+2 will only use the new field, which was set in 0.N+1.